### PR TITLE
fix documentation now that the worker and the webui are split

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

**Comment by coolo** [comment]
I also removed backlog and perl subdirs - we don't use them for ages

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -217,20 +217,22 @@ The openQA web interface can be started via +MOJO_REVERSE_PROXY=1 morbo script/o
 development mode.
 
 +/var/lib/openqa/+ must be owned by root and contain several sub
-directories owned by the user that runs the workers (default 'geekotest'):
+directories owned by the user that runs openqa (default 'geekotest'):
 
-* +backlog+
 * +cache+ (cache directory, needs to be owned by the user who runs openqa)
 * +factory/iso+
 * +perl/+ (subdirectory 'autoinst' must link to +/usr/lib/os-autoinst+)
```
Comment (aaannz) [inline docs/Installing.asciidoc]: You removed perl subdir too, right?

File: docs/Installing.asciidoc [hunk 2]
```diff
@@ -217,20 +217,22 @@ The openQA web interface can be started via +MOJO_REVERSE_PROXY=1 morbo script/o
...
-* +pool/1+ (add more for each worker)
 * +share+ (shared directories for remote workers)
 * +testresults+
 * +tools+ (symlink to +/usr/share/openqa/tools/+)
+* ++db+ contains the sqlite database
```
Comment (aaannz) [inline docs/Installing.asciidoc]: extra + in prefix

File: docs/Installing.asciidoc [hunk 3]
```diff
@@ -217,20 +217,22 @@ The openQA web interface can be started via +MOJO_REVERSE_PROXY=1 morbo script/o
...
 
-+/var/lib/openqa/db+ contains the sqlite database and must be owned by
-the user that runs the web interface ('geekotest'). To initially create this
-database, you need to run +tools/initdb+.
+To initially create the database, you need to run +script/initdb+.
+
+The worker needs to own +/var/lib/openqa/pool/$INSTANCE+, e.g.
```
Comment (aaannz) [inline docs/Installing.asciidoc]: Workers can create their instance directories so maybe some info that owning /var/lib/openqa/pool is enough for worker self management?

File: docs/Installing.asciidoc (hunk 1 above)
Comment (coolo) [inline docs/Installing.asciidoc]: right

File: docs/Installing.asciidoc (hunk 3 above)
Comment (coolo) [inline docs/Installing.asciidoc]: /var/lib/openqa/pool is owned by root in the package, so possibly we should discuss how agile we want our workers to be.

**Comment by coolo** [comment]
see 0c494f5

**Pull request merged**